twentytwentyone-blocks/experimental-theme.json

```diff
@@ -136,6 +136,35 @@
 						"name": "Gigantic"
 					}
 				],
+				"fontWeights": [
+					{
+						"slug": "light",
+						"value": 300,
+						"name": "Light"
+					},
+					{
+						"slug": "normal",
+						"value": "normal",
+						"name": "Normal"
+					},
+					{
```

Comment:
You should also add a `600` semibold weight in here — it's used for [Headings](https://github.com/WordPress/twentytwentyone/blob/30379d158e67cd33d02ecd0b68970cfb4609b7c9/assets/sass/01-settings/global.scss#L55) and [pagination](https://github.com/WordPress/twentytwentyone/blob/30379d158e67cd33d02ecd0b68970cfb4609b7c9/assets/sass/01-settings/global.scss#L184).